Source/WTF/wtf/Assertions.h

```diff
@@ -248,16 +248,58 @@ WTF_EXPORT_PRIVATE void WTFReleaseLogStackTrace(WTFLogChannel*);
 
 WTF_EXPORT_PRIVATE bool WTFIsDebuggerAttached(void);
 
+#if (OS(DARWIN) || PLATFORM(PLAYSTATION)) && (CPU(X86_64) || CPU(ARM64))
```

Comment:
I think this `#if` is messing up the wpe build. I don't think we need it anyway.